Atomic/scan.py

```diff
@@ -193,6 +204,11 @@ def get_additional_args():
             json_results = json.load(open(json_file))
             if json_results['Successful'].upper() != 'TRUE':
                 return 1
+
+        # Perform remediation
+        for i in self.scan_list:
+            self.remediate(self.remediation_script, i.id, self.results_dir)
```

Comment:
This should be run only if `--remediate` was specified.

Reply:
Fixed. Nice catch